frontend/catalyst/api_extensions/control_flow.py

```diff
@@ -309,80 +309,7 @@ def loop_rx(i, x):
     """
 
     def _body_query(body_fn):
```

Comment:
Personally I would love it if this wasn't called `_body_query`, something `decorator` or `wrapper` is much more common.

Reply:
done